src/main/java/Duke.java

```diff
@@ -1,10 +1,81 @@
+import java.util.Scanner;
+import java.util.ArrayList;
+
 public class Duke {
     public static void main(String[] args) {
```

Comment:
Perhaps you can abstract out some parts of the main() function into separate functions with descriptive function names for greater readability, as main() currently seems really long and hard to understand